Sleek/color.ini

```diff
@@ -1,25 +1,25 @@
 [Wealthy]
 ; Green on dark grey background
-text               = 8bc34a
-subtext            = b4b4b4
-button-text        = 202020
-main               = 202020
-sidebar            = 202020
-player             = 242424
-card               = 242424
-shadow             = 000000
-selected-row       = 2a3c17
-sub-button         = 6a913d
-button             = 8bc34a
-button-active      = 98da4b
-button-disabled    = 353535
-sidebar-button     = 8bc34a
-play-button        = 8bc34a
-tab-active         = 303030
-notification       = 242424
-notification-error = 242424
-playback-bar       = 8bc34a
-misc               = FFFFFF
+text               = 8bc34a ;
+subtext            = b4b4b4 ;
+button-text        = 202020 ;
+main               = 202020 ;
+sidebar            = 202020 ;
+player             = 242424 ;
+card               = 242424 ;
+shadow             = 000000 ;
+selected-row       = 2a3c17 ;
+sub-button         = 6a913d ;
+button             = 8bc34a ;
+button-active      = 98da4b ;
+button-disabled    = 353535 ;
+sidebar-button     = 8bc34a ;
+play-button        = 8bc34a ;
+tab-active         = 303030 ;
+notification       = 242424 ;
+notification-error = 242424 ;
+playback-bar       = 8bc34a ;
+misc               = FFFFFF ;
```

Comment:
Why did you add semicolon at the end?

Reply:
srry it was for my own comment but forgot to remove it before upload